app/src/main/java/org/mozilla/reference/browser/topsites/ui/TopSitesFeature.kt

```diff
@@ -4,22 +4,20 @@
 
 package org.mozilla.reference.browser.topsites.ui
 
+import android.content.Context
 import mozilla.components.browser.icons.BrowserIcons
-import mozilla.components.feature.session.SessionUseCases.LoadUrlUseCase
-import mozilla.components.feature.tabs.TabsUseCases
 import mozilla.components.support.base.feature.LifecycleAwareFeature
 import org.mozilla.reference.browser.library.history.usecases.HistoryUseCases
 
 class TopSitesFeature(
+    private val context: Context,
     private val topSitesView: TopSitesView,
-    private val loadUrlUseCase: LoadUrlUseCase,
-    private val tabsUseCases: TabsUseCases,
     private val historyUseCases: HistoryUseCases,
     private val browserIcons: BrowserIcons
 ) : LifecycleAwareFeature {
 
     override fun start() {
-        topSitesView.init(loadUrlUseCase, tabsUseCases, historyUseCases, browserIcons)
+        topSitesView.init(context, historyUseCases, browserIcons)
```

Comment:
Please rename `init` to `initialize`. A view doesn't need a `Context`, it has already one associated.